logger/logger.go

```diff
@@ -3,15 +3,19 @@ package logger
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
-	"log"
 	"net/http"
 	"os"
 	"regexp"
+	"strings"
 	"time"
+
 	"github.com/joho/godotenv"
 )
 
+var defaultIngestURL = "https://logs.logdna.com/logs/ingest"
```

Comment:
Maybe outside the scope of this PR, but we should break this into the two parts for domain and endpoint since we have different endpoints and different domains.

Reply:
yeah good point, lets consider that separately. we do stick to a single configuration option in the latest node client, and im not sure there is a scenario where the endpoint would be changed? i could see the config just being `IngestDomain` instead but we should be consistent across clients wherever possible.

Reply:
The only case I can think of is logging to the supertenant endpoint? But I'm not 100% sure if we want to surface that in our clients.